purchase_request/views/purchase_request_line_view.xml

```diff
@@ -449,12 +449,6 @@
                                 readonly="1"
                             />
                             <field name="date_planned" widget="date" readonly="1" />
-                            <field
```

Comment:
Why removing this?

Reply:
Because the company field does not add any value, it can only confuse.

Reply:
I don't agree. It serves for knowing the company of the record.

Reply:
Ok, so we keep it, it doesn't make sense IMO but I'll undo it.